app/controllers/settings/exports/muted_accounts_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module Settings
   module Exports
-    class MutedAccountsController < ApplicationController
+    class MutedAccountsController < BaseController
```

Comment:
ok, if this inherits from Settings::BaseController, doesn't that still require_functional!?